test/util.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+# This utility file contains functions that wrap commands to be tested and allow us to interface 
+# our command-line tests with JUnit on Jenkins. All wrapper functions run commands in a sub-shell 
+# and redirect all output. Tests in test-cmd *must* use these functions for testing.
+
+# HIGH-LEVEL WRAPPER FUNCTIONS: ASSUMING EXIT CODES
+
+# expect_success runs the cmd and expects an exit code of 0
+function os::test::expect_success() {
```

Comment:
Nearly every single script that we have is for testing.  We don't have other scripts.  `os::test::` is providing almost zero entropy, at least `os::cmd::` tells me this is about commands.